fusor-ember-cli/app/routes/review/progress/overview.js

```diff
@@ -32,8 +33,42 @@ export default Ember.Route.extend({
     controller.set('rhevTask', model.rhevTask);
     controller.set('openstackTask', model.openstackTask);
     controller.set('cfmeTask', model.cfmeTask);
+    controller.set('deployment', model.deployment);
     controller.stopPolling();
-    controller.startPolling();
+
+    ////////////////////////////////////////////////////////////
+    // NOTE: If an error during a pulp sync occurs, the Katelo::Sync
```

Comment:
@eriknelson, I don't recall why I had `stopPolling` before `startPolling`
```
    controller.stopPolling();
    controller.startPolling();
```
However, are you sure you want to remove `startPolling` in the `setupController`?  If so, what is now starting the polling for the progress bars?